subxt/src/blocks/extrinsic_types.rs

```diff
@@ -558,6 +607,117 @@ impl<T: Config> ExtrinsicEvents<T> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct ExtrinsicSignedExtensions<'a> {
+    bytes: &'a [u8],
+    metadata: Metadata,
+}
+
+#[derive(Debug, Clone)]
+pub struct ExtrinsicSignedExtension<'a> {
+    bytes: &'a [u8],
+    ty_id: u32,
+    identifier: &'a str,
+    metadata: Metadata,
+}
+
+impl<'a> ExtrinsicSignedExtensions<'a> {
+    /// Get a certain signed extension by its name.
+    pub fn find(&self, signed_extension: impl AsRef<str>) -> Option<ExtrinsicSignedExtension> {
```

Comment:
To be consistent with how Extrinsics/Events are decoded, I think `find` should look more like:

```rust
fn find<S: SignedExtension>() -> Result<Option<S>, Error>
```

For "dynamically finding things", one can `.iter().find(..)` anyway.

I'd suggest just removing this for now until we have a nice way to find and decode based on the `subxt::tx::signed_extensions::*` types.